cmd/thor/httpserver/api_server.go

```diff
@@ -142,15 +142,14 @@ func StartAPIServer(
 	if config.Timeout > 0 {
 		router.Use(middleware.HandleAPITimeout(time.Duration(config.Timeout) * time.Millisecond))
 	}
-
 	router.Use(handlers.CompressHandler)
```

Comment:
In the previous version this was `handler := handlers.CompressHandler(router)`. Is this producing the same output?